pkg/manager/runtime_env/help.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
```

Comment:
I think there is no problem. Please change the file name to `helper.go`, and then rebase your commits.